installbuilder/GNUmakefile

```diff
@@ -56,7 +56,7 @@ all:
 		--RUN_AS_USER="$(RUN_AS_USER)" \
 		--OUTPUTFILE=$(PACKAGE_NAME) \
 		--DATAFILE_PATH="$(TOP)/scriptprovider/installbuilder" \
-		Base_OMIScriptProvider.data Linux.data Linux_DPKG.data
+		Base_OMIScriptProvider.data Linux.data Linux_RPM.data
 
 	@echo "========================= Make DSC installer"
```

Comment:
You're not making DSC installer. But/paste error. Please fix this here and above at 35.

Other cut/paste errors?